### DISCUSSION TITLE
Always on top toggles - in menu, on toolbar, as a hotkey

### DISCUSSION THREAD

**Discussion by fbck** [Chatterino/chatterino2#3311] [Ideas]
It would be convenient to have a way to switch Always on top option in a more easily accessible way than opening Settings and switching it there.
It could be placed as an option in the menu, as it's own button next to the other buttons, and it could have it's own hotkey (in program or global - it could be convenient to have a hotkey that would toggle and bring up the window.)

**Comment by Mm2PL** [comment]
This could kinda qualify for the sequel of #2340 as editing settings. But I'm not so sure about the global shortcut part, making these is a pain and you have to repeat it for all platforms the program runs on. You can use your OS's scripting capabilities or AutoHotkey on Windows.
